mods/climate/temperature.lua

```diff
@@ -243,7 +243,9 @@ local adjust_active_temp = function(pos, temp)
    --Underground
    if pos.y < av_temp_depth then
       --average temp, heating under the earth. ~ 25C/km
-      temp = -0.025*pos.y + av_temp
+      --calculate depth factor based on temp of 40 at -1000
+      local av_depth_factor = (40 - av_temp) / (-1000-av_temp_depth)
+      temp = av_depth_factor*pos.y + av_temp
```

Comment:
This will probably cause a big jump when you hit the av_temp_depth level, as you're going from applying 0 depth factor when pos.y => avt_depth, to applying pos.y*factor one step down, which could be -12 to -70*factor depending on the starting depth.
It should be factor*(pos.y - avt_depth) + av_temp